### PR TITLE
Added background image

### PR DESCRIPTION
**Pull request by Addisu87**
I have added a background image for the mobile version.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -12,7 +12,7 @@ body {
 }
 
 .container {
-  background-image: url('images/header-shape-desktop.png');
+  background-image: url('images/background-mobile.png');
   background-repeat: no-repeat;
```
Comment (RolandM99) [inline style.css:16]: - [ ] Good job implementing some requests from the previous review, but you didn't yet fit correctly the original design from the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
### Table below:
| Your design  | Figma design |
|  -------------   | ------------- |
| image | image |

Kindly, make sure that your website looks exactly like the original design. The font, style, color, and margin should be the same. Please, implement your changes referencing the Figma design you choose.

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @Addisu87 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._
- [ ] Please, kindly make sure that your PR is descriptive. For this project, I can see that you created a new PR to ask for a re-review(because I'm your previous reviewer). But, this is not allowed. You have to submit the one a previous reviewer reviewed so that your next reviewer can see what you have asked to improve. For the next review, you should submit this PR, you don't need to create a new one.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Addisu87) [reply style.css:16]: @RolandM99 How can i move up the background image. I can fit as the Figma design.

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @Addisu87 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**